ui/admin-portal/src/app/components/util/candidate-search-card/candidate-search-card.component.ts

```diff
@@ -89,6 +82,10 @@ export class CandidateSearchCardComponent implements OnInit, AfterViewChecked {
     return this.candidate.selected;
   }
 
+  get isSubmissionList(): boolean {
+    return this.isList && this.candidateSource.sfJobOpp != null
```

Comment:
I don't think that this is a good use of "get". This is method that performs some logic and returns a value. It is not accessing a property.

If you remove the "get" you need to change the method call in the html - see comment above.